docs/media-buy/targeting.md

```diff
@@ -166,12 +149,9 @@ Media buys can refine the targeting:
 }
 ```
 
-### 3. Final Effective Targeting
+### Final Effective Targeting
 
-The system combines both layers:
-- Lists are merged (union for `any_of`, union for `none_of`)
-- Structured fields use the most specific value
-- Custom fields are merged by platform
+The targeting overlay is applied directly to the media buy. The implementation_config in the product contains ad server-specific settings, while the targeting overlay controls audience selection.
```

Comment:
target overlay could also control ad server settings like geo - the ad server setup would be the combination of both